primedev/mods/modmanager.cpp

```diff
@@ -628,6 +619,28 @@ void ModManager::SearchFilesystemForMods()
 	std::sort(m_LoadedMods.begin(), m_LoadedMods.end(), [](Mod& a, Mod& b) { return a.LoadPriority < b.LoadPriority; });
 }
 
+void ModManager::ExportModsConfigurationToFile()
+{
+	m_EnabledModsCfg.SetObject();
```

Comment:
Does `m_EnabledModsCfg.SetObject();` have to be set again here?

(I don't think I fully understand what that line does... ^^)

Reply:
It resets the state of the `m_EnabledModsCfg` object (same as `dict = {}` in js for instance)